.travis.yml

```diff
@@ -0,0 +1,39 @@
+---
+# Adapted from
+# - https://github.com/drhelius/travis-ansible-demo/blob/master/.travis.yml
+# - https://github.com/geerlingguy/ansible-role-docker/blob/master/.travis.yml
+language: python
+services:
+  - docker
+
+env:
+  global:
+    - ANSIBLE_ROLE: dokku_bot.ansible_dokku
+
+matrix:
+  include:
+    - name: Ubuntu-18.04
+      env:
+        - DOCKER_IMAGE: marvelnccr/ubuntu-docker-base
```

Comment:
Can you include the Dockerfile for this image? Or maybe we start a new dokku repo for managing that base image.